modules/mod_auth.c

```diff
@@ -2435,15 +2435,6 @@ MODRET auth_user(cmd_rec *cmd) {
   origuser = user;
   c = pr_auth_get_anon_config(cmd->tmp_pool, &user, NULL, NULL);
 
-  /* Check for AccessDenyMsg */
```

Comment:
Addresses:
```
mod_auth.c:2443:7: warning: Value stored to 'denymsg' is never read
      denymsg = sreplace(cmd->tmp_pool, denymsg, "%u", user, NULL);
      ^         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Turns out this `AccessDenyMsg` check here is never used; probably leftover from earlier code.